src/graphql/__tests__/joke.test.ts

```diff
@@ -38,6 +38,19 @@ beforeAll(() => {
 });
 
 describe('query.joke', () => {
+  it('should return a result with errors if query is incorrect', async () => {
+    const query = `
+      query Q {
+        jokeByCategory {
+          nonExistentProperty
+        }
+      }
+    `;
+
+    const result = await graphql(schema, query, rootValue, context);
+    expect(result.errors).toBeDefined();
```

Comment:
Sorry for the long response time. Maybe a simple solution would be to just store the whole result as a snapshot?

Reply:
+1, this is what we've been doing in the graphql projects for a while and it seems to work well for most error scenarios (e.g. typo in the query)

Reply:
example:
```
    Received value does not match stored snapshot 1.

    - Snapshot
    + Received

      Object {
    -   "data": Object {
    -     "jokeByCategory": Object {
    -       "categories": Array [
    -         "NERDY",
    -       ],
    -       "id": "1000",
    -       "text": "Mock Chuck Norris's database has only one table, 'Kick', which he DROPs frequently.",
    -     },
    -   },
    -   "errors": undefined,
    +   "data": undefined,
    +   "errors": Array [
    +     [GraphQLError: Cannot query field "idd" on type "Joke". Did you mean "id"?],
    +   ],
      }

      70 |     const { data, errors } = result;
      71 |
    > 72 |     expect({ data, errors }).toMatchSnapshot();
      73 |   });
      74 | });
      75 |
```